cfme/middleware/__init__.py

```diff
@@ -1,8 +1,39 @@
-from cfme.web_ui import CheckboxTable, toolbar as tb
 from functools import partial
 
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils.browser import ensure_browser_open
+from cfme.common import Validatable, SummaryMixin
+from utils.log import logger
+
+
 cfg_btn = partial(tb.select, 'Configuration')
 mon_btn = partial(tb.select, 'Monitoring')
 pol_btn = partial(tb.select, 'Policy')
+pwr_btn = partial(tb.select, 'Power')
+
+
+def list_tbl():
+    return CheckboxTable(table_locator="//div[@id='main-content']//div[@id='list_grid']/table")
```

Comment:
Unless you do version picking on the page objects, it is pretty pointless to keep it in a function.

```python
list_tbl = CheckboxTable(table_locator="//div[@id='main-content']//div[@id='list_grid']/table")
```

Besides, you have two ID locators in your xpath. Unless developers did something wrong and list_grid is defined multiple times, you should use only the list_grid.